project.clj

```diff
@@ -15,7 +15,8 @@
                                      [commons-io/commons-io "2.4"]]}
              :1.5    {:dependencies [[org.clojure/clojure "1.5.1"]]}
              :1.7    {:dependencies [[org.clojure/clojure "1.7.0"]]}
-             :master {:dependencies [[org.clojure/clojure "1.8.0-master-SNAPSHOT"]]}}
+             :1.11    {:dependencies [[org.clojure/clojure "1.11.1"]]}
+             :master {:dependencies [[org.clojure/clojure "1.8.0"]]}}
```

Comment:
Bump to 1.11 ?

Reply:
not sure what backwards compat. plans for the lib are so I rather did not change this